features/networking/multicast.feature

```diff
@@ -273,3 +275,344 @@ Feature: testing multicast scenarios
       | <%= cb.proj2pod1ip %>.*multicast, xmt/rcv/%loss = 5/5/0%                |
       | <%= cb.proj2pod2ip %>.*multicast, xmt/rcv/%loss = 5/5/0%                |
     """
+
+  # @author hongli@redhat.com
```

Comment:
@weliang1 you submitted the PR but the `@author is honglin@rehdat.com`??

Reply:
> @weliang1 you submitted the PR but the `@author is honglin@rehdat.com`??

Here are the what this PR does:
1. Copy all multicast cases from cucushift to verification-tests, now all the multicast cases can be run from verification-tests
2.Update YAML path to use <%= BushSlicer::HOME %>/testdata/networking/multicast-rc.json
3. Add more steps in OCP-12966 to wait old pods are deleted in default name before creating the new pods

So you will see @author is still honglin@rehdat.com for those old test cases. There are no new test cases created in the PR

Reply:
@weliang1 is any special reason copy these scenario to verification-tests repo?

Reply:
> @weliang1 is any special reason copy these scenario to verification-tests repo?

@zhaozhanqi The main reason to run all multicast case from one repo is to update, maintain and debug those cases easily, we really do not want spread same feature scripts in two repo locations. We discussed this in our sync up meeting several weeks ago, the multicast is upstream feature and it is fine to run all multicast cases from verfication-tests rep

Reply:
@weliang1 I can presume following implications if we move these tests to public repo

- You need to generate two PRs , against master and v3 branch of verification-tests repo. 
  This PR points to master only.
- To keep  ```Given /^the env is using multitenant or networkpolicy network``` compatible, it 
  either needs a bypass logic for master branch PR like 
  https://github.com/openshift/verification-tests/pull/1096#issuecomment-628899201 
  or it can be removed from networking.rb in ```master``` branch and hence removed from 
  multicast scenarios as well in `master` branch PR
- Update all multicast scripts in Polarion to point to right location after it s moved